fvm/src/machine/default.rs

```diff
@@ -24,11 +25,18 @@ use crate::kernel::{ClassifyResult, Result};
 use crate::machine::limiter::DefaultMemoryLimiter;
 use crate::machine::Manifest;
 use crate::state_tree::{ActorState, StateTree};
-use crate::syscall_error;
 use crate::system_actor::State as SystemActorState;
+use crate::{syscall_error, EMPTY_ARR_CID};
 
 pub const EVENTS_AMT_BITWIDTH: u32 = 5;
 
+lazy_static::lazy_static! {
+    /// The CBOR-serialized bytes of the empty array
+    pub static ref EMPTY_ARRAY_BLOCK_DATA: Vec<u8> = {
```

Comment:
I wanted to have this be a `Block`, but that requires specifying the trait type `D: AsRef<[u8]> + ?Sized`, which led to Rust being Rust.

Reply:
You should be able to specify `D` as `Vec<u8>`?

Reply:
Done!